addons/laser/functions/fnc_onLaserDesignatorDraw.sqf

```diff
@@ -15,14 +15,19 @@
  * Public: No
  */
 
-private _laserCode = ACE_player getVariable[QGVAR(code), ACE_DEFAULT_LASER_CODE];
+params ["_ctrl"];
+
+private _display = ctrlParent _ctrl;
+
+private _laserCode = ACE_player getVariable QGVAR(code);
```

Comment:
```suggestion
private _laserCode = ACE_player getVariable[QGVAR(code), ACE_DEFAULT_LASER_CODE];
```
otherwise it displays the config text `text = "Code: 1001";` which isn't actually a valid code 😄